packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -18,7 +19,13 @@ import BrowserDetection from '../common/browser-detection';
 
 const {isBrowser} = BrowserDetection();
 
-type MultistreamConnectionConfig = ConstructorParameters<typeof MultistreamRoapMediaConnection>[0];
+// TODO: in internal-media-core add appropriate types in MultistreamConnectionConfig for enable main audio and video
```

Comment:
I don't understand this todo comment, but anyway, enableMainAudio and enableMainVideo have been removed from MultistreamConnection constructor parameters in WCME 3.0, so any references to them should be removed also from internal-media-core (I can see they're still there) and from here and from Media.createMediaConnection function

Reply:
@marcin-bazyl - This was added after the tests failed because of this. I think particularly this test [here](https://github.com/webex/webex-js-sdk/commit/9a313e115ed59d2baacb7e4b4c935c142136c5e0#diff-826c5d0df79137a1fb8dba26f75550d10afa643de27ee68684256200af661a17R226).

Looks like it checks if MultistreamRoapMediaConnection's constructor is called with `enableMainAudio` and `enableMainVideo`

So, I thought those are missed out when creating types. Do we want to remove that check from the test instead?

Reply:
Yes, the test needs to be updated. As you can se [here](https://sqbu-github.cisco.com/WebExSquared/webrtc-media-core/blob/8705925929ce9084b43416ad4682e23f1e14284e/src/MediaConnection/config.ts#L48) the MultistreamRoapMediaConnection's constructor takes MultistreamConnectionConfig that doesn't have these fields anymore (you've removed them in this [PR](https://sqbu-github.cisco.com/WebExSquared/webrtc-media-core/pull/192/) )

Reply:
Sorry. These were cherry-picked changes from Bryce and so missed to notice. Thanks. Will update thet tests accordingly

Reply:
Done in #3053